plasma_framework/contracts/poc/fast_exits/Quasar.sol

```diff
@@ -272,13 +272,17 @@ contract Quasar is QuasarPool {
      * @param rlpChallengeTx RLP-encoded challenge transaction
      * @param challengeTxInputIndex index pos of the same utxo in the challenge transaction
      * @param challengeTxWitness Witness for challenging transaction
+     * @param sharedOutputInputIndex (optional) index pos of another input from the claimTx that is spent
```

Comment:
`sharedOutputInput` seems confusing to me 😛 
We're talking about an input to the claim tx that's not the exiting output, right?
How about
`sharedInputIndex` and `sharedInputCreationTx `
or 
`otherInputIndex` and `otherInputCreationTx`

Not sure what's the best name, feel free to leave it as it is if you think it's clear like that

Reply:
Yeah 😅, It really looks confusing. I'll change it to `otherInputIndex` and `otherInputCreationTx`